src/wr_control_drive_science/config/scienceControl.yaml

```diff
@@ -0,0 +1,2 @@
+fileLoc: "placeholder"
```

Comment:
For added clarity, add a comment about how this should be a `/dev/ttyUSB` file for initial setup.